Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0-alpine3.15 AS build-env
+
+ARG MS_NUGET_URL=https://nuget.pkg.github.com/microsoft/index.json
+ARG MSGRAPH_NUGET_URL=https://nuget.pkg.github.com/microsoftgraph/index.json
+ARG NUGET_USER
```

Comment:
https://docs.docker.com/engine/reference/builder/#arg

> It is not recommended to use build-time variables for passing secrets like github keys, user credentials etc. Build-time variable values are visible to any user of the image with the docker history command.

Reply:
Oh...I'll update this.

Reply:
sure, but the secrets appear in the log and might leak into the CI logs (github actions or ADO pipelines) because of that.

Reply:
unresolving this as this is still a concern

Reply:
We can turn off logging on CI